charts/fybrik/notebook-test-readflow.tls.values.yaml

```diff
@@ -3,7 +3,7 @@
 # It overrides values that are different from the default values.yaml
 # The values used for testing read flow where the manager and connectors
 # uses mutual tls connection. In addition, OPA server is configured to use
-# mutual tls.
+# mutual tls and Vault server is configured to use tls.
```

Comment:
mutual TLS is the server setting, how and where do you configure Vault to require mTLS ?

Reply:
@roytman I changed the test to use Vault with mutual TLS. please also see manager/testdata/notebook/README.txt for test description. Thanks